src/webprojects/fxa.md

```diff
@@ -0,0 +1,116 @@
+# Firefox Accounts (FxA): how to localize and test
+
+Firefox Accounts (or FxA) refers to https://accounts.firefox.com/signin. Every month, tens of millions of active internet users have logged in to a Firefox account, of which, more than half of the users worldwide choose localized versions. Firefox Accounts lets you access to a slew of Firefox products and services through the Firefox browsers on desktop and mobile devices: Lockwise, Pocket, Monitor, Screenshots, Send, and Sync. By using the same account, you can log into [Pontoon](https://pontoon.mozilla.org/projects/) and [SuMo](https://support.mozilla.org) to start contributing to localization.
+
+There are three components in Pontoon that make up [FxA](https://pontoon.mozilla.org/projects/firefox-accounts/):
+
+* `LC_MESSAGES/client.po`: the content server for UI and frontend
+* `LC_MESSAGES/server.po`: strings for emails and backend templates
+* `Main.ftl`: payments for both frontend and backend
+
+The above could change over time as the team is planning to migrate .po files to .ftl, which may involve restructuring the files.
+
+## Key links
+
+* Production: https://accounts.firefox.com/
+* Staging: https://latest.dev.lcip.org/
+* Repository: https://github.com/mozilla/fxa-content-server-l10n
+* Pontoon: https://pontoon.mozilla.org/de/firefox-accounts/
+
+## Localizing the strings
+
+FxA lives in two file formats.  The shorter a string is, especially the kind containing variable or placeable, the more context you need to make sure you get the translation right. Look for clues or ask others for clarification:
+
+* String comments: Pay attention to the comment or string ID. In Pontoon, the comments are displayed above the area where you can enter your translation for a string. Some strings are used in more than one place. Finding the appropriate translation that works broadly is important.
+* [Matrix channel](https://chat.mozilla.org/#/room/#l10n-community:mozilla.org): ping the L10n PM in charge with any issues you may have.
+* [Web projects mailing list](https://groups.google.com/g/mozilla.dev.l10n.web): Raise questions here so others may benefit from the information.
+
+## Breaking the code, breaking the build
```

Comment:
Forgot this: it's not the localizers' fault if the code is brittle. "Errors in translations can break the build for all languages"